src/metrics.ts

```diff
@@ -10,14 +10,6 @@ export const metricDeclarations = {
     type: IMetricsComponent.CounterType,
     labelNames: ['pathname']
   },
-  dcl_ws_rooms: {
```

Comment:
Don't we need these two?

Reply:
dcl_ws_rooms_connections aready exists in rooms.ts and dcl_ws_rooms is replaced by dcl_ws_rooms_count in rooms.ts